contracts/NXMDSValue.sol

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) 2020 NexusMutual.io
+
+  This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/ */
+
+pragma solidity 0.5.7;
+
+
+contract Aggregator {
+    function currentAnswer() public view returns (uint); 
+}
+
+
+contract NXMDSValue {
+
+    /// @dev Gets DAI feed address from chainlink feed and converts into desired format.
+    /// @return It return DAI Feed rate in 10^18 format.  
+    function read() public view returns (bytes32)
+    {
+        
+        // Instance to get DAI feed from chainlink feed.
+        Aggregator aggregator = Aggregator(0x79fEbF6B9F76853EDBcBc913e6aAE8232cFB9De9);
```

Comment:
don't hard code the address, have it as a parameter instead

Reply:
I don't think we should make it configurable because we will have to give the right to change the address to owner/someone else and it will make it out of scope of NXM governance. Instead, if anything needs to be changed here in future, we can easily deploy a new contract and change the DAI feed address via NXM governance. This makes it more transparent and less vulnerable. Does that make sense?

Reply:
ok, let's leave this as is and go for deploying a new contract if changes are required.